<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>News_page</class>
 <widget class="QDialog" name="News_page">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>774</width>
    <height>626</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Dialog</string>
  </property>
  <widget class="QListView" name="listView">
   <property name="geometry">
    <rect>
     <x>130</x>
     <y>120</y>
     <width>411</width>
     <height>291</height>
    </rect>
   </property>
  </widget>
  <widget class="QPushButton" name="allNewsButton">
   <property name="geometry">
    <rect>
     <x>10</x>
     <y>20</y>
     <width>161</width>
     <height>29</height>
    </rect>
   </property>
   <property name="text">
    <string>Display All News</string>
   </property>
  </widget>
  <widget class="QPushButton" name="dateButton">
   <property name="geometry">
    <rect>
     <x>500</x>
     <y>20</y>
     <width>171</width>
     <height>29</height>
    </rect>
   </property>
   <property name="text">
    <string>Display News by date</string>
   </property>
  </widget>
  <widget class="QComboBox" name="categorycombox">
   <property name="geometry">
    <rect>
     <x>240</x>
     <y>50</y>
     <width>161</width>
     <height>25</height>
    </rect>
   </property>
  </widget>
  <widget class="QLabel" name="label">
   <property name="geometry">
    <rect>
     <x>180</x>
     <y>50</y>
     <width>51</width>
     <height>21</height>
    </rect>
   </property>
   <property name="text">
    <string>Choose:</string>
   </property>
  </widget>
  <widget class="QLabel" name="label_2">
   <property name="geometry">
    <rect>
     <x>230</x>
     <y>20</y>
     <width>171</width>
     <height>20</height>
    </rect>
   </property>
   <property name="text">
    <string>Display News by category</string>
   </property>
  </widget>
  <widget class="QPushButton" name="pushButton_spam">
   <property name="geometry">
    <rect>
     <x>550</x>
     <y>210</y>
     <width>201</width>
     <height>121</height>
    </rect>
   </property>
   <property name="text">
    <string>Choose and Push if Spam</string>
   </property>
  </widget>
  <widget class="QPushButton" name="ratebutton">
   <property name="geometry">
    <rect>
     <x>500</x>
     <y>60</y>
     <width>171</width>
     <height>29</height>
    </rect>
   </property>
   <property name="text">
    <string>Display News by rate</string>
   </property>
  </widget>
  <widget class="QLabel" name="label_3">
   <property name="geometry">
    <rect>
     <x>150</x>
     <y>460</y>
     <width>91</width>
     <height>20</height>
    </rect>
   </property>
   <property name="text">
    <string>Choose title:</string>
   </property>
  </widget>
  <widget class="QLabel" name="label_4">
   <property name="geometry">
    <rect>
     <x>180</x>
     <y>510</y>
     <width>41</width>
     <height>20</height>
    </rect>
   </property>
   <property name="text">
    <string>Rate:</string>
   </property>
  </widget>
  <widget class="QComboBox" name="titlecombox">
   <property name="geometry">
    <rect>
     <x>260</x>
     <y>460</y>
     <width>151</width>
     <height>25</height>
    </rect>
   </property>
  </widget>
  <widget class="QComboBox" name="ratingcombox">
   <property name="geometry">
    <rect>
     <x>270</x>
     <y>500</y>
     <width>75</width>
     <height>25</height>
    </rect>
   </property>
   <item>
    <property name="text">
     <string>1</string>
    </property>
   </item>
   <item>
    <property name="text">
     <string>2</string>
    </property>
   </item>
   <item>
    <property name="text">
     <string>3</string>
    </property>
   </item>
   <item>
    <property name="text">
     <string>4</string>
    </property>
   </item>
   <item>
    <property name="text">
     <string>5</string>
    </property>
   </item>
  </widget>
  <widget class="QPushButton" name="ratinpushbutton">
   <property name="geometry">
    <rect>
     <x>370</x>
     <y>500</y>
     <width>93</width>
     <height>29</height>
    </rect>
   </property>
   <property name="text">
    <string>Enter</string>
   </property>
  </widget>
  <widget class="QLabel" name="label_5">
   <property name="geometry">
    <rect>
     <x>140</x>
     <y>550</y>
     <width>121</width>
     <height>20</height>
    </rect>
   </property>
   <property name="text">
    <string>Go to comments:</string>
   </property>
  </widget>
  <widget class="QPushButton" name="commentsbtn">
   <property name="geometry">
    <rect>
     <x>270</x>
     <y>550</y>
     <width>93</width>
     <height>29</height>
    </rect>
   </property>
   <property name="text">
    <string>comments</string>
   </property>
  </widget>
  <widget class="QPushButton" name="logoutButton">
   <property name="geometry">
    <rect>
     <x>640</x>
     <y>580</y>
     <width>93</width>
     <height>29</height>
    </rect>
   </property>
   <property name="text">
    <string>Log out</string>
   </property>
  </widget>
 </widget>
 <resources/>
 <connections>
  <connection>
   <sender>ratinpushbutton</sender>
   <signal>clicked()</signal>
   <receiver>ratingcombox</receiver>
   <slot>clearEditText()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>425</x>
     <y>477</y>
    </hint>
    <hint type="destinationlabel">
     <x>351</x>
     <y>473</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>ratinpushbutton</sender>
   <signal>clicked()</signal>
   <receiver>titlecombox</receiver>
   <slot>clearEditText()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>440</x>
     <y>482</y>
    </hint>
    <hint type="destinationlabel">
     <x>385</x>
     <y>444</y>
    </hint>
   </hints>
  </connection>
 </connections>
</ui>
